scvid/transforms/transforms.py

```diff
@@ -0,0 +1,28 @@
+import torch
+from torch import nn
+
+
+class LogNormalize(nn.Module):
+    """
+    LogNormalize gene counts with mean, standard deviation, and pseudo-count.
```

Comment:
I belive I was sloppy with me description.  In the following expression:
$log(1 + C \times x_{ng} / \sum_g x_{ng})$

The pseudo-count is 1, and C, is fixed target total count (not a pseudo-count). Let's do the following:

- Rename `C` to `target_count`
- Make scaling by `std_g` optional via a `scale` boolean flag.
- If applicable, add `eps: float = 1e-6` (a new __init__ arg) to `std_g` in the denominator to obviate division by zero.
-

Reply:
Or maybe have `std_g` as `Optional[torch.Tensor] = None` and not apply it if it is None?